spec/requests/regions_spec.rb

```diff
@@ -9,23 +9,15 @@
 #
 
 RSpec.describe "Regions API", :regions do
+  let(:region) { FactoryBot.create(:miq_region, :region => '2') }
```

Comment:
This is probably invalid, technically, as the id will not be part of that region's range.  However, since this was a copy/paste from below, it's ok for now.